frameit/lib/frameit/device_types.rb

```diff
@@ -1,11 +1,13 @@
 module Frameit
   module Color
-    BLACK = "SpaceGray"
-    SILVER = "Slvr"
+    BLACK = "Space Gray"
```

Comment:
@tadelv I'm still thinking: how would you want to select if you want an iPhone 7 or an iPhone 6? The screenshots are the same size